### PR TITLE
changes to create account and restore them in multiple networks

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

Bugfixes to get multiple accounts working

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmSelectNetwork.vue [hunk 1]
```diff
@@ -46,9 +46,10 @@ export default {
     sortedNetworks() {
       // sorts networks setting showing the current network first, mainnets at the top and the default one the first
       if (this.networks.length > 0) {
+        const selectedNetwork = this.networks.find(({ id }) => id === this.networkId)
         return [
           // current network first
-          this.networks.find(({ id }) => id === this.networkId),
+          ...(selectedNetwork ? [selectedNetwork] : []),
```
Comment (faboweb) [inline src/components/common/TmSelectNetwork.vue]: why this complicated syntax?

File: src/components/common/TmSessionImportName.vue [hunk 2]
```diff
@@ -94,16 +94,29 @@ export default {
       set(value) {
         this.$store.commit(`updateField`, { field: `name`, value })
       }
+    },
+    prefix: {
```
Comment (faboweb) [inline src/components/common/TmSessionImportName.vue]: nope, the prefix should be got from the networkId. please check how the createKey function in keystore.js works

File: src/components/common/TmSelectNetwork.vue (hunk 1 above)
Comment (iambeone) [reply src/components/common/TmSelectNetwork.vue]: We can use the extension without the networkId set. And in this case we are getting an error in previous code.

Comment (faboweb) [reply src/components/common/TmSelectNetwork.vue]: instead why not use `.filter(x => !!x)` to filter out undefineds?

Comment (faboweb) [reply src/components/common/TmSelectNetwork.vue]: I will create a PR for that

Comment (faboweb) [reply src/components/common/TmSelectNetwork.vue]: https://github.com/luniehq/lunie/pull/3493

File: src/vuex/modules/recover.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ const emptyState = {
   seed: ``,
   password: ``,
   passwordConfirm: ``,
+  prefix: ``,
```
Comment (faboweb) [inline src/vuex/modules/recover.js]: remove

File: src/components/common/TmSessionImportPassword.vue [hunk 4]
```diff
@@ -118,7 +118,8 @@ export default {
           seedPhrase: this.recover.seed,
           password: this.recover.password,
           name: this.recover.name,
-          network: this.networkId
+          network: this.networkId,
+          prefix: this.recover.prefix
```
Comment (faboweb) [inline src/components/common/TmSessionImportPassword.vue]: remove

**Comment by Bitcoinera** [comment]
I also just extensively tested this PR and it is working like a charm!

Great work @iambeone ! clap clap clap

Will now look into those failing tests

**Pull request merged**